styles.css

```diff
@@ -13,29 +10,76 @@ body { font-family: 'Roboto', sans-serif; }
   justify-content: space-between;
 }
 
-.header__logo { padding: 40px 0; }
+.logo {
+  padding: 40px 0;
+  font: bold 24px/31px 'pt_serif', arial, helvetica, sans-serif;
+  padding-left: 44px;
+  position: relative;
+}
+
+.logo__circle {
+  width: 28px;
+  height: 28px;
+  position: absolute;
+  transform: translateY(-50%);
+  top: 50%;
+  left: 0;
+  background: #0056ec;
+  opacity: .7;
+  border-radius: 50%;
+}
+
+.logo__circle_color_blue {
+left: 8px;
+top: 56%;
+background: #c8dbfb;
+opacity: .7;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html_2.0/pullRequest?prid=6308834)